package-lock.json

```diff
@@ -319,30 +319,30 @@
       }
     },
     "node_modules/@bcrs-shared-components/business-lookup": {
-      "version": "1.3.4",
-      "resolved": "https://registry.npmjs.org/@bcrs-shared-components/business-lookup/-/business-lookup-1.3.4.tgz",
-      "integrity": "sha512-R4RrMp23NX5f9JJuTHW0YjYVMLJMsty9aElmRbKCPrLznR2DWj8MkFX4wwQVPTBdsMXS7K5WdD98ob3SNdK4WA==",
+      "version": "1.3.15",
+      "resolved": "https://registry.npmjs.org/@bcrs-shared-components/business-lookup/-/business-lookup-1.3.15.tgz",
+      "integrity": "sha512-YMf6hK7IJYGC5ykuhJxJhI1YmCcKTx1JQvpqaBhR0lqQVBnPIuxtL9MTBZ+lsAQyuTf12S4H04BjQ+rYavG/Kg==",
       "dependencies": {
-        "@bcrs-shared-components/interfaces": "^1.1.15",
+        "@bcrs-shared-components/interfaces": "^1.1.25",
         "lodash": "4.17.21",
         "vue": "^2.7.14"
       }
     },
     "node_modules/@bcrs-shared-components/business-lookup/node_modules/@bcrs-shared-components/enums": {
-      "version": "1.1.11",
-      "resolved": "https://registry.npmjs.org/@bcrs-shared-components/enums/-/enums-1.1.11.tgz",
-      "integrity": "sha512-/19lQ/DLo0hS0HWVNhhHqsjnp3Ed8f7g60kStQGFni6mOm1AHaEtSot9rUs56msYncmw4BNY6ho1o3ZvIeHlHQ==",
+      "version": "1.1.20",
+      "resolved": "https://registry.npmjs.org/@bcrs-shared-components/enums/-/enums-1.1.20.tgz",
+      "integrity": "sha512-H91adgNbIHLKcaujZdrdTFEGgDxcdHYcD4564OLSBrf+gXWE6TRT1A58pmOCCC3a8zWhXKn+TECqumc5/YMRSA==",
       "dependencies": {
         "@bcrs-shared-components/corp-type-module": "^1.0.16"
       }
     },
     "node_modules/@bcrs-shared-components/business-lookup/node_modules/@bcrs-shared-components/interfaces": {
-      "version": "1.1.16",
-      "resolved": "https://registry.npmjs.org/@bcrs-shared-components/interfaces/-/interfaces-1.1.16.tgz",
-      "integrity": "sha512-RxmR+6W6t0lRkDGVb4VEHfEqgK76pk42LaJn36aSrd/hNIMfxgHEAqPT21zBJ5on5rvWzFMBemp3hV6DyUTw8Q==",
+      "version": "1.1.25",
```

Comment:
Ditto for interfaces.

Watch for any new code warnings or errors (in case an interface changed in a meaningful way) -- or compare code changes like you did before.